src/main/scala/net/snowflake/spark/snowflake/io/StageWriter.scala

```diff
@@ -697,7 +697,18 @@ private[io] object StageWriter {
             DefaultJDBCWrapper.resolveTable(conn, table.name, params)
           if (list.isEmpty || list.get.isEmpty) {
             ConstantString("(") + tableSchema.fields
-              .map(_.name)
+              .map(
```

Comment:
To reviewer:
1. Apply the same quotation strategy like ELSE clause.
2. Reuse existing internal option `params.quoteJsonFieldName` to disable the column name quote if necessary.